nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/pom.xml

```diff
@@ -69,6 +69,11 @@
             <version>2.6.6</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
```

Comment:
Does it need to be added in nifi-aws-nar's [NOTICE](https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-aws-bundle/nifi-aws-nar/src/main/resources/META-INF/NOTICE) ?